package.json

```diff
@@ -1253,6 +1253,7 @@
     "rxjs-marbles": "^5.0.6",
     "sass-loader": "^10.3.1",
     "selenium-webdriver": "^4.4.0",
+    "sharp": "^0.30.7",
```

Comment:
not the perfect solution (for png, jpg is mostly equivalent) but we're already pulling it in with synthetics, but the package is maintained and performance is good.